generate_wordstr_box.py

```diff
@@ -44,6 +48,10 @@ def trim(im):
 # create WordStr line boxes for Indic & RTL
 for line in lines:
     line = unicodedata.normalize('NFC', line.strip())
+    if args.rtl:
+        # FIXME: This should not be necessary. Compare with e.g. kraken
+        line = line.translate(str.maketrans("()[]{}»«><", ")(][}{«»<>"))
+        line = bidi.algorithm.get_display(line)
     if line:
         print("WordStr 0 0 %d %d 0 #%s" % (width, height, line))
```

Comment:
```suggestion
        print("WordStr 0 0 %d %d 0 #%s" % (width, height, bidi.algorithm.get_display(line)))
```

Reply:
I think we don't need a new option and can call `bidi.algorithm.get_display(line)` unconditionally here.